src/types/index.d.ts

```diff
@@ -132,10 +132,10 @@ export interface FacetOption extends Record<string, any> {
 }
 
 export interface Group extends BaseGroup, Record<string, any> {
-  count: number;
-  data: Record<string, any>;
-  parents: BaseGroup[];
-  children: Group[];
+  count?: number;
+  data?: Record<string, any>;
+  parents?: BaseGroup[];
+  children?: Group[];
```

Comment:
Modified to follow the [Docs on Groups](https://docs.constructor.io/rest_api/search/groups_structure/) more accurately. I checked w/ Islam and this change doesn't affect any existing usage of this type as they were meant to be wrapped in `Partial<>` when used.